src/domains/ecommerce-app/api/ecommerce-transactions-service/v2/_openapi.json.tpl

```diff
@@ -41,6 +41,15 @@
         ],
         "description": "Create a new transaction activating the payments notice by meaning of 'Nodo' ActivatePaymentNotice primitive",
         "parameters": [
+          {
+            "in": "header",
+            "name": "X-Client-Id",
+            "required": true,
```

Comment:
required?
it is not required in https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/470